retrieval/target.go

```diff
@@ -197,10 +197,24 @@ func (t *Target) Update(cfg *config.ScrapeConfig, baseLabels, metaLabels clientm
 
 	t.url.Scheme = cfg.Scheme
 	t.url.Path = string(baseLabels[clientmodel.MetricsPathLabel])
+	params := url.Values{}
+	for k, v := range cfg.Params {
+		params[k] = make([]string, len(v))
+		copy(params[k], v)
+	}
+	for k, v := range baseLabels {
+		if strings.HasPrefix(string(k), "__param_") {
```

Comment:
Alright, since the client change is merged now, this will require a re-vendoring to then use the constant here.

@beorn7 are we ok with vendoring non-released versions of client_golang between Prometheus releases, or do you prefer them to always be released ones (IMO ok to use unrealesed ones for non-release versions of Prometheus)?